music/views.py

```diff
@@ -1,3 +1,38 @@
 from django.shortcuts import render
+from django.views.generic import View
+from .forms import SearchForm
+from .models import Track, Album, Artist, Genre, Playlist
+from django.http import HttpResponse
 
-# Create your views here.
+class IndexView(View):
+    template_name = 'index.html'
+    form = SearchForm
+
+    def post(self, request):
+        search = request.POST['search']
+        form = SearchForm(request.POST)
+        if Track.objects.filter(track_name=search):
```

Comment:
No need to conditionally check for tracks, albums and artists. Instead, you should search in all three of the models and pass them in a context and render once